src/Analyzer/Pass/Statement/ReturnVoid.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace PHPSA\Analyzer\Pass\Statement;
+
+use PhpParser\Node\Stmt;
+use PhpParser\Node;
+use PhpParser\Node\Stmt\Return_;
+use PHPSA\Analyzer\Helper\DefaultMetadataPassTrait;
+use PHPSA\Analyzer\Pass;
+use PHPSA\Context;
+
+class ReturnVoid implements Pass\AnalyzerPassInterface
+{
+    use DefaultMetadataPassTrait;
+
+    const DESCRIPTION = 'Checks for return void statements.';
+
+    /**
+     * @param $stmt
+     * @param Context $context
+     * @return bool
+     */
+    public function pass(Return_ $stmt, Context $context)
```

Comment:
As I known on PHP 7.1 We are having :void as return type
I think We should check it
Can you add it?